spec/spec.md

```diff
@@ -623,10 +626,12 @@ DID by its type.
   - `id` - **string** - **REQUIRED** - ID of the DID to resolve.
 - **Returns:**
   - `200` - Success.
-    - `did` - **object** - A JSON object representing the DID Document.
-    - `types` - **array** - An array of [type strings](#type-indexing) for the DID.
-    - `sequence_numbers` - **array** - An sorted array of seen sequence numbers, used with [historical resolution](#historical-resolution).
-  - `400` - Invalid request.
+    - `did` - **object** - **REQUIRED** - A JSON object representing the DID Document.
+    - `pkarr` - **string** - **REQUIRED** - An unpadded base64URL-encoded representation of the full [[ref:Pkarr]] payload, represented as 64 bytes sig,
+    - `types` - **array** - **OPTIONAL** - An array of [type integers](#type-indexing) for the DID.
+    - `sequence_numbers` - **array** - **OPTIONAL** - An sorted array of seen sequence numbers, used with [historical resolution](#historical-resolution).
+    8 bytes u64 big-endian seq, 0-1000 bytes of v concatenated; enabling independent verification.
+    - `400` - Invalid request.
```

Comment:
I think the `8 bytes u64 big-endian...` text should appear after the `- pkarr` line:

```suggestion
    - `did` - **object** - **REQUIRED** - A JSON object representing the DID Document.
    - `pkarr` - **string** - **REQUIRED** - An unpadded base64URL-encoded representation of the full [[ref:Pkarr]] payload, represented as 64 bytes sig,
    8 bytes u64 big-endian seq, 0-1000 bytes of v concatenated; enabling independent verification.
    - `types` - **array** - **OPTIONAL** - An array of [type integers](#type-indexing) for the DID.
    - `sequence_numbers` - **array** - **OPTIONAL** - An sorted array of seen sequence numbers, used with [historical resolution](#historical-resolution).
    - `400` - Invalid request.
```